test/Libraries/DynamoPythonTests/PythonEngineSelectorTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using Dynamo;
+using NUnit.Framework;
+using PythonNodeModels;
+
+namespace DynamoPythonTests
+{
+    [TestFixture]
+    class PythonEngineSelectorTests: DynamoModelTestBase
+    {
+        protected override void GetLibrariesToPreload(List<string> libraries)
+        {
+            // Add multiple libraries to better simulate typical Dynamo application usage.
+            libraries.Add("DSCPython.dll");
+            libraries.Add("DSIronPython.dll");
+            base.GetLibrariesToPreload(libraries);
+        }
+
+        /// <summary>
+        /// This test will cover the initial state of the Singleton
+        /// </summary>
+        [Test]
+        public void TestEngineSelectorInitial_State()
+        {
+            Assert.AreEqual(false, PythonEngineSelector .lazy.IsValueCreated);
+            Assert.AreEqual(false, PythonEngineSelector .IsCPythonEnabled);
+            Assert.AreEqual(false, PythonEngineSelector .IsIronPythonEnabled);
```

Comment:
Strange place to put a space. Was it intended?

Reply:
ah these comes with the class renaming, will fix